src/eureka/S1_detector_processing/s1_process.py

```diff
@@ -163,7 +163,12 @@ def run_eurekaS1(self, filename, meta, log):
         - February 2022 Aarynn Carter /  Eva-Maria Ahrer
             Updated for JWST version 1.3.3, code restructure
         '''
-        # Run the pipeline
+        # Define superbias offset procedure
```

Comment:
What's the reason for moving this up from where it was before?

Reply:
That was to resolve #521 by moving the `self.superbias = Eureka_SuperBiasStep()` line earlier